src/js/components/Box/doc.js

```diff
@@ -212,7 +212,9 @@ export const doc = Box => {
       PropTypes.string,
     ]).description(`The amount of spacing between child elements. This
         should not be used in conjunction with 'wrap' as the gap elements
-        will not wrap gracefully.`),
+        will not wrap gracefully. If a child component is or renders a fragment,
+        Box will not add a gap between the individual elements in the
+        fragment`),
```

Comment:
```suggestion
        fragment.`),
```

Reply:
I can word it that way if you prefer. I did specifically choose that phrasing because there is a difference between a functional component that returns a fragment and a literal fragment as a child. The latter we could actually detect and iterate over it's children from what I can tell

Reply:
updated